datatables.go

```diff
@@ -133,16 +133,23 @@ func (p *Parser) bindTeamScores() {
 
 			if s != nil {
 				// Set values that were already updated
-				s.id = teamID
-				s.clanName = clanName
-				s.flag = flagImage
-				s.score = score
+				s.ID = teamID
+				s.ClanName = clanName
+				s.Flag = flagImage
+				s.Score = score
 
 				// Register direct updates for the future
 				// Except for teamId, it doesn't change; players swap teams instead
-				entity.BindProperty("m_szClanTeamname", &s.clanName, st.ValTypeString)
-				entity.BindProperty("m_szTeamFlagImage", &s.flag, st.ValTypeString)
-				entity.BindProperty("m_scoreTotal", &s.score, st.ValTypeInt)
+				entity.BindProperty("m_szClanTeamname", &s.ClanName, st.ValTypeString)
+				entity.BindProperty("m_szTeamFlagImage", &s.Flag, st.ValTypeString)
+
+				entity.FindProperty("m_scoreTotal").OnUpdate(func(val st.PropertyValue) {
+					s.Score = val.IntVal
+
+					p.eventDispatcher.Dispatch(events.ScoreUpdated{
+						TeamState: s,
+					})
```

Comment:
Hmm. I wonder if we want to dispatch this event on 0-0 score as well?

Reply:
Good point. We should also consider the case of a forced score-reset though (so we don't go from 1-0 to 0-1 without an event for 0-0) e.g. if the match wasn't live yet etc. etc.

Reply:
I just checked and I think this can be ignored. `OnUpdate()` isn't triggered for the initial `0` value.